Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertColumn.cs

```diff
@@ -151,10 +153,14 @@ public ApplicationObject ColumnToNative(Column speckleColumn)
 
         //don't change offset for slanted columns, it's automatic
         if (!isLineBased)
-          SetOffsets(revitColumn, speckleRevitColumn, level, topLevel);
+          SetOffsets(revitColumn, level, topLevel, ScaleToNative(speckleRevitColumn.baseOffset, speckleRevitColumn.units), ScaleToNative(speckleRevitColumn.topOffset, speckleRevitColumn.units));
 
         SetInstanceParameters(revitColumn, speckleRevitColumn);
       }
+      else
+      {
+        SetOffsets(revitColumn, level, topLevel, -baseOffset, -topOffset);
```

Comment:
I think we want to add a check here for if (!isLineBased) similar to how we're doing it on line 155. We don't want to call this method for lineBased (aka slanted) columns

Reply:
Yes, you are right. The column import didn't work, because there were no baseLine-s exported from Archicad. Now it is added (but only the vertical case, I'm about to implement the slanted cases, but in a separate PR).
Because all the Columns coming from not Revit are base line defined, I just simple removed the code you are referring. I tested, the columns top and bottom elevations are set correctly.